test/models/type-alias-event2.ts

```diff
@@ -0,0 +1,21 @@
+import * as mongoose from 'mongoose';
+
+import { prop, Typegoose } from '../../src/typegoose';
+
+export class Event extends Typegoose {
+  // constructor(fieldTwo: string, fieldOne?: string) {
```

Comment:
why is this code commented-out and still in here?

Reply:
Oh sorry, I'm going to remove it